source/configure/configuration-settings.rst

```diff
@@ -4279,6 +4278,37 @@ Token
 
 Generate a token to validate incoming requests from AWS SNS by selecting ``Regenerate``.
 
+Calls
+~~~~~
+
+Access the following configuration settings in the System Console by going to **Plugins > Calls**.
+
+Enable Plugin
+^^^^^^^^^^^^^
+
+|all-plans| |self-hosted| |cloud|
+
+**True**: Enables the calls plugin on your Mattermost workspace.
+
+**False**: Disables the calls plugin on your Mattermost workspace.
+
+**RTC Server Port**: Default setting is 8443.
+
+**Enable on Specific Channels**: This can be set to **true** or **false**.
+
+**Enable on All Channels**: This can be set to **true** or **false**.
+
+**ICE Host Override**: This is an optional field.
+
+**ICE Servers**: This is an optional field.
+
+**RTCD Service URL**: This is an optional field.
+
+Experimental Features
```

Comment:
@streamer45, I didn't see this option in the screenshot I was referencing. Is this an option on this screen? I may need an updated screenshot.

Reply:
I am not aware of any "Experimental Features" section. This is the latest:

![image](https://user-images.githubusercontent.com/1832946/172842365-f35f8d8e-7dff-4ef8-bd46-52307e7f80ea.png)

Reply:
Ah awesome - the one I had was missing the max call participants. Thanks.